tools/sql/clitest/versionTest.go

```diff
@@ -185,22 +185,23 @@ func (s *VersionTestSuite) runCheckCompatibleVersion(
 	s.NoError(err)
 	port := strconv.Itoa(mysqlPort)
 
-	s.NoError(sql.RunTool([]string{
+	err = sql.RunTool([]string{
```

Comment:
it's outside this diff-scope so for GitHub Reasons™ I can't comment directly on it but:

would it be easy enough to move this to a `t.TempDir()` rather than a manual tempdir?  That'd remove the need to clean up.

Reply:
fixed